pwa/src/components/componentCard/ComponentCard.module.css

```diff
@@ -98,3 +106,8 @@
 .dataCategory {
   --conduction-tag-background-color: var(--web-app-layer-category-color-data);
 }
+
+.unknownLayer {
+  --conduction-tag-background-color: var(--web-app-color-top-nav-grey);
```

Comment:
I don't think a `...-top-nav-grey` token should be used here. Two issues:

1. This is not a `top-nav`, so using a dedicated token for another component is not logical and can create issues later on (e.g. if the `top-nav` is later rebranded to be orange).
2. Giving such a specific name to a token, kind of beats te purpose of the token. Right now it _has_ to hold a `grey` value. What if someone wants to make a orange top navigation? The key should simply be `..color-top-nav`, and even more specific, it should be `-background-color-top-nav`.